pkg/renderer/html5/_macros/hello

```diff
@@ -0,0 +1,13 @@
+{{- if eq .Kind "block" -}}
```

Comment:
FYI, in https://github.com/bytesparadise/libasciidoc/issues/349 I've suggested that we have define the `inline` and `block` macros separately, as I believe that it's simpler to manage for the users (no need to change it in here for now, but that may be refactored in the future)

Reply:
I think so too.